<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="song_link">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>25</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="placeholderText">
    <string>                    Song Link</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>370</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Song : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>85</y>
     <width>370</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Artist :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="process_button">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>25</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Process</string>
   </property>
  </widget>
  <widget class="QPushButton" name="download_button">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>25</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>370</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>File Size :</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
